features/contract-state/Totals.tsx

```diff
@@ -64,8 +64,19 @@ const Totals = () => {
     tokenAddress !== null &&
     provider !== null
   ) {
-    const prettyTotalCollateral = Number(totalCollateral).toLocaleString();
-    const prettyTotalTokens = Number(totalTokens).toLocaleString();
+    const prettyTotalCollateral = Number(totalCollateral).toLocaleString(
```

Comment:
this fixes a wrap with the number of ren tokens created.

before:
![image](https://user-images.githubusercontent.com/12886084/92477808-86386580-f1e1-11ea-93d2-5a86e352c8a5.png)

after:
![image](https://user-images.githubusercontent.com/12886084/92477822-8d5f7380-f1e1-11ea-9a80-d38aa57d0c9a.png)